gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadChannel.java

```diff
@@ -1060,6 +1060,42 @@ protected InputStream openStream(long bytesToRead) throws IOException {
     }
   }
 
+  private void cacheFooterWithRetries(Storage.Objects.Get getObject, HttpResponse response)
+      throws IOException {
+    for (int retriesCount = 0; retriesCount < maxRetries; retriesCount++) {
```

Comment:
Usually we retry with exponential backoff. Since this is nice to have, can we make this simpler by not retrying?